src/loaders/express/session.ts

```diff
@@ -1,25 +1,25 @@
-import connectMongo from 'connect-mongo'
+import MongoStore from 'connect-mongo'
 import cookieParser from 'cookie-parser'
-import session, { SessionOptions } from 'express-session'
+import session from 'express-session'
 import { Connection } from 'mongoose'
 
-import config from '../../config/config'
-
-const sessionMongoStore = connectMongo(session)
+import config, { nodeEnv } from '../../config/config'
+import { Environment } from '../../types/config'
 
 const sessionMiddlewares = (connection: Connection) => {
   // Configure express-session and connect to mongo
   const expressSession = session({
     saveUninitialized: false,
     resave: false,
     secret: config.sessionSecret,
-    // TODO(#42): Remove the typecast once `config` has correct types.
-    cookie: config.cookieSettings as SessionOptions['cookie'],
+    cookie: config.cookieSettings,
     name: 'connect.sid',
-    store: new sessionMongoStore({
-      mongooseConnection: connection,
-      collection: 'sessions',
-    }),
+    store:
+      nodeEnv === Environment.Test
+        ? new session.MemoryStore()
+        : MongoStore.create({
+            clientPromise: Promise.resolve(connection.getClient()),
+          }),
```

Comment:
should we create a test-specific sessionMiddleware, and only leave the `MongoStore.create` part here?